frontend/packages/process-editor/src/components/Canvas/Canvas.tsx

```diff
@@ -38,6 +41,11 @@ export const Canvas = ({ onSave, appLibVersion }: CanvasProps): JSX.Element => {
     onSave(await getUpdatedXml());
   };
 
+  useConfirmationDialogOnPageLeave(
+    !!numberOfUnsavedChanges,
```

Comment:
What do you think about the following suggestion:
```suggestion
    Boolean(numberOfUnsavedChanges),
```